README.md

```diff
@@ -1,494 +1,301 @@
+    - path: /etc/sysupdate.d/noop.conf
       contents:
         inline: |
-          <redacted>
+          [Source]
+          Type=regular-file
+          Path=/
+          MatchPattern=invalid@v.raw
+          [Target]
+          Type=regular-file
+          Path=/
   links:
-    - target: /opt/extensions/wasmcloud/wasmcloud-1.2.1-x86-64.raw
-      path: /etc/extensions/wasmcloud.raw
+    - target: /opt/extensions/EXTNAME/EXTNAME-3.13.5-x86-64.raw
+      path: /etc/extensions/EXTNAME.raw
       hard: false
 systemd:
   units:
-    - name: nats.service
-      enabled: true
-      dropins:
-        - name: 10-nats-env-override.conf
-          contents: |
-            [Service]
-            Environment=NATS_CONFIG=/etc/nats-server.conf
-    - name: wasmcloud.service
-      enabled: true
-      dropins:
-        - name: 10-wasmcloud-env-override.conf
-          contents: |
-            [Service]
-            Environment=WASMCLOUD_LATTICE=<redacted>
     - name: systemd-sysupdate.timer
       enabled: true
     - name: systemd-sysupdate.service
       dropins:
-        - name: wasmcloud.conf
-          contents: |
-            [Service]
-            ExecStartPre=/usr/lib/systemd/systemd-sysupdate -C wasmcloud update
-        - name: sysext.conf
+        - name: EXTNAME.conf
           contents: |
             [Service]
-            ExecStartPost=systemctl restart systemd-sysext
+            ExecStartPre=/usr/bin/sh -c "readlink --canonicalize /etc/extensions/EXTNAME.raw > /tmp/EXTNAME"
+            ExecStartPre=/usr/lib/systemd/systemd-sysupdate -C EXTNAME update
+            ExecStartPost=/usr/bin/sh -c "readlink --canonicalize /etc/extensions/EXTNAME.raw > /tmp/EXTNAME-new"
+            ExecStartPost=/usr/bin/sh -c "if ! cmp --silent /tmp/EXTNAME /tmp/EXTNAME-new; then <SERVICE REFRESH ACTION HERE>; fi"
 ```
 
-This example uses Butane/Ignition configuration do the following customizations beyond simply including the image:
+Note that this snippet inlines all of the sysupdate configuration for documentation purposes.
+Most extensions published in the Bakery ship sysupdate configuration as part of the releases, so these can be downloaded by Ignition instead of being inlined.
+(And consequently, that's what most of the configuration examples of individual extensions do).
+Check out the respective extensions' readme for details.
 
-1. Provide a different configuration to setup the nats-server to act as a leaf node to a pre-existing wasmCloud deployment (`/etc/nats-server.conf`).
-2. Provide a set of credentials for the nats-server leaf node to connect with (`/etc/nats.creds`).
-3. Override the bundled `NATS_CONFIG` environment variable to point it to the newly created configuration (`NATS_CONFIG=/etc/nats-server.conf`).
-4. Override the lattice the wasmCloud host is configured to connect (`WASMCLOUD_LATTICE=<redacted>`).
+Also, we include a dummy `noop.conf` for systemd-sysupdate to work around a spurious error message.
```

Comment:
it's unclear whether the user would need to add a similar clause/only required temporarily/only for the sake of this example. maybe add a remark to provide context?

Reply:
I've never _seen_ the issue but have carried along this workaround for the time being. Kai added this some time ago, might be cargo cult :shrug: